server/config/config.go

```diff
@@ -31,9 +32,20 @@ func (c *Config) SetMinGasPrices(gasPrices sdk.DecCoins) {
 // GetMinGasPrices returns the validator's minimum gas prices based on the set
 // configuration.
 func (c *Config) GetMinGasPrices() sdk.DecCoins {
-	gasPrices, err := sdk.ParseDecCoins(c.MinGasPrices)
-	if err != nil {
-		panic(fmt.Sprintf("invalid minimum gas prices: %v", err))
+	if c.MinGasPrices == "" {
+		return sdk.DecCoins{}
+	}
+
+	gasPricesStr := strings.Split(c.MinGasPrices, ";")
+	gasPrices := make(sdk.DecCoins, len(gasPricesStr))
```

Comment:
? Wouldn't it be better to have a `NewDecCoins(...)` constructor kind of thing?

Reply:
I don't think so.